src/Plugin/GraphQL/Schema/SdlSchemaPluginBase.php

```diff
@@ -198,6 +202,10 @@ protected function getExtensionDocument(array $extensions = []) {
       return !empty($definition);
     });
 
+    if (empty($ast)) {
```

Comment:
This seems unnecessary and also wrong because $ast is not even defined there yet!

Reply:
Its needed other wise you have a parse error unexpected end of file. and change $ast to $extensions